### PR TITLE
Add carusel for cars

### PR DESCRIPTION
**Pull request by Gegardus**
In this part of the project, I put react pagination on the main page so that the user can see a list of cars that are selected as a theme.

### CODE REVIEW

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Gegardus 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

**You have submitted a project that has **conflicts** with the default branch. Kindly make sure you fix them bore requesting a new review. I know you can do it :muscle: :muscle:** 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by DeVuDeveloper — COMMENTED** [review]
### **Hi @Gerardus,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/cars/CarsCarousel.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+];
+
+const cars = [
+  { car: 'https://flyclipart.com/thumb2/aston-martin-cars-png-images-free-download-12690.png' },
+  { car: 'https://www.nicepng.com/png/detail/64-648130_southwest-custom-classic-cars-all-new-sport-cars.png' },
+  { car: 'https://png.pngtree.com/png-clipart/20210722/ourmid/pngtree-car-blue-transportation-reception-sports-car-png-image_6561592.png' },
+  { car: 'https://png.pngtree.com/png-clipart/20210722/ourmid/pngtree-car-red-trolley-sports-car-png-image_6561572.png' },
+];
+
+const CarsCarousel = () => {
```
Comment (DeVuDeveloper) [inline src/components/cars/CarsCarousel.js:20]: Good job. but in mobile view paginator will not show all cars

**Comment by gerardus** [comment]
Hi, I think that you are sending those messages to the wrong email.

File: src/components/cars/CarsCarousel.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  const pageCount = Math.ceil(cars.length / carsPerPage);
+
+  const changePage = ({ selected }) => {
+    setPageNumber(selected);
+  };
+
+  return (
+    <div className="flex flex-col gap-4 mt-16 lg:flex-row">
+      {displayCars.map((item) => (
+        <div className="flex flex-col items-center w-80 bg-green-200" key={item.name}>
```
Comment (DeVuDeveloper) [inline src/components/cars/CarsCarousel.js:34]: we should follow design strictly and bg collor green is not nessesery here.. single car should be wrapped with one circle and car should take background of thet div and with should be more than 100 %

File: src/components/cars/Paginate.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactPaginate from 'react-paginate';
+
+const Paginate = ({ pageCount, changePage }) => (
```
Comment (DeVuDeveloper) [inline src/components/cars/Paginate.js:5]: This is good wotk but we need cars form API, kindly fetch cars from API with image, car_model and description and implement it

File: src/components/cars/Paginate.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    previousLinkClassName="previous-btn"
+    nextLinkClassName="next-btn"
+    disabledLinkClassName="disable-btns"
+  />
+);
+
+Paginate.propTypes = {
+  changePage: PropTypes.func.isRequired,
+  pageCount: PropTypes.number.isRequired,
+};
```
Comment (DeVuDeveloper) [inline src/components/cars/Paginate.js:21]: branch has conflicts, follow instructions and fetch first data from develop branch, resolve conflicts and then doPR

**Review by DeVuDeveloper — CHANGES_REQUESTED** [review]

File: src/components/cars/CarsCarousel.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          <img src={item.car} alt="car" className="car rounded-full w-60 h-60 blob" />
+          <h3 className="uppercase text-slate-900 text-lg mt-6 font-bold">
+            Details
+          </h3>
+          <hr className="border-t-2 border-dashed border-gray-500 w-32 mt-4" />
+          <p className="text-gray-400 text-center font-semibold mt-4">{item.details}</p>
+          <div className="flex gap-4 mt-4">
+            {social.map((item) => (
```
Comment (DeVuDeveloper) [inline src/components/cars/CarsCarousel.js:42]: good job, but pls implement links to social media

**Review by DeVuDeveloper — CHANGES_REQUESTED** [review]

File: src/components/cars/CarsCarousel.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+import facebook from '../../assets/images/fb-icon-home.png';
+
+const social = [
+  { icon: facebook },
+  { icon: twitter },
+  { icon: insta },
+];
+
+const cars = [
+  { car: 'https://flyclipart.com/thumb2/aston-martin-cars-png-images-free-download-12690.png' },
```
Comment (DeVuDeveloper) [inline src/components/cars/CarsCarousel.js:14]: When user click on image should be redirected to car details page

**Review by DeVuDeveloper — CHANGES_REQUESTED** [review]

**Pull request closed without merging**